CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## Unreleased
+
+### Fixed
+
+- Fixed `LoggerConnector` to have logged metrics on root device in DP ([#4138](https://github.com/PyTorchLightning/pytorch-lightning/pull/4138))
+
```

Comment:
should this be under rc?

Reply:
but the rc tag was already pushed?

Reply:
All from now till final 1.1 shall go here and we just update section header as all between 1.0.8 and 1.1 shall be one section
Btw, in past we had also rc and chlogs in the save section...

Reply:
ok, so that means I need to move it to the rc section?

Reply:
Ok remove RC from the heading...
The rc will be later also removed from pypi and release... 

The same as we always